README.md

```diff
@@ -234,31 +225,36 @@ Go to the [SDK Documentation](https://lightclient.raiden.network/docs/) for more
 
 Go to the [CLI README](https://github.com/raiden-network/light-client/tree/master/raiden-cli) for installation instructions and the current status of the API.
 
-### dApp Installation
+### Install and Run dApp
 
-```bash
-git clone --recurse-submodules https://github.com/raiden-network/light-client.git
-cd light-client/raiden-dapp
-```
+1. Clone repository  
 
-#### Build the Raiden SDK
+   ```bash
+   git clone --recurse-submodules https://github.com/raiden-network/light-client.git
+   cd light-client
+   ```
 
-First you need to build the sdk. For this you have to go to the `raiden` directory and run the following commands.
+2. Install dependencies  
 
-```bash
-pnpm install
-pnpm run build --filter raiden-ts
-```
+   ```bash
+   yarn install
+   ```
 
-#### Running the dApp locally
+3. Build the Raiden SDK  
 
-To start the development server you have to run the following command.
+   ```bash
+   cd raiden-ts
+   yarn build
```

Comment:
I'd suggest replacing all these `cd <folder>` commands with the respective workspaces command:
```suggestion
   yarn workspace raiden-ts build
```
This way, we remove the _which folder are we in now_ variable of the instructions.